modules/ocf_www/files/build-vhosts

```diff
@@ -353,8 +349,12 @@ def main():
         report(config)
         return 0
 
-    changed = test_and_overwrite_config(site_cfg, config, args.target)
+    if args.target == 'web':
+        config_test_cmd = ('apachect', 'configtest')
```

Comment:
s/apachect/apachectl/

Reply:
Ah, that explains the error email I saw earlier

Reply:
That's I get for hasty refactoring...